### ISSUE TITLE
Host links on html_fancy page are broken

### ISSUE THREAD

**Issue by keepwatch** [fboender/ansible-cmdb#85]
There is a typo that breaks name URLs on the html_fancy page. URLs in the generated HTML look like the following:
```
  <a href="#hostname["name"]}">hostname</a>
</td>
```
The typo is in [html_fancy.tpl](https://github.com/fboender/ansible-cmdb/blob/master/src/ansiblecmdb/data/tpl/html_fancy.tpl#L43-L45):
```
<%def name="col_name(host)">
  <a href="#${jsonxs(host, 'name')}["name"]}">${jsonxs(host, "name")}</a>
</%def>
```
Note the duplicate 'name' reference - `"#${jsonxs(host, 'name')}["name"]}"`. This results in a URL that looks like `#hostname[`, the part between the first two quotes. [This pull request](https://github.com/fboender/ansible-cmdb/pull/84) simply removes the legacy `["name"]}` code so that these links work as normal.

**Comment by fboender** [comment]
Pull request has been merged. Thank you!

**Comment by arturmartins** [comment]
Thanks @keepwatch! It is quite an annoying bug.

**Comment by fboender** [comment]
Sorry about this everyone, I had no idea this problem existed. I've released a new version of ansible-cmdb: [v1.16](https://github.com/fboender/ansible-cmdb/releases/tag/1.16)
